readme.md

```diff
@@ -1136,19 +1136,22 @@ Expressions can be empty or contain just a comment:
 All content (headings, paragraphs, etc) you write are exported as the default
 export from a compiled MDX file as a component.
 
-It’s possible to pass components in.
+It’s possible to pass props in. The special prop `components` is used to
+determine how to render components.  This includes both JSX and markdown syntax.
 Say we have a `message.mdx` file:
 
 ```mdx
 # Hello, *<Planet />*!
+
+Remember when we first met {props.year}?
 ```
 
 This file could be imported from JavaScript and passed components like so:
 
 ```js
 import Message from './message.mdx' // Assumes an integration is used to compile MDX -> JS.
 
-<Message components={{Planet: () => 'Venus'}} />
+<Message components={{Planet: () => 'Venus'}} year={1962} />
```

Comment:
That year is the first time human spacecrafts visited Venus right?

Reply:
Yes, according to Wikipedia.